change/@griffel-webpack-extraction-plugin-b5b23fb4-5c39-4a06-a877-b9f9a7a33ba1.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

```suggestion
  "type": "patch",
```

As this thing is still experimental, let's not bump it to `1.0.0`.